src/libsystemd/sd-event/sd-event.c

```diff
@@ -2095,7 +2095,7 @@ _public_ int sd_event_add_inotify(
                 sd_event_inotify_handler_t callback,
                 void *userdata) {
 
-        sd_event_source *s;
+        sd_event_source *s = NULL;
```

Comment:
We usually add `/* avoid false maybe-uninitialized warning */` comment in such cases other places. Please also add it here.

Reply:
done.